website/blog/2021-03-05-GAAD-React-Native-Accessibility.md

```diff
@@ -0,0 +1,33 @@
+---
+title: The GAAD Pledge - Improving React Native Accessibility
+author: Alexandra Marlette
+authorTitle: React Native Accessibility Open Source Community Manager
+authorURL: 'https://twitter.com/alexmarlette'
+authorImageURL: 'https://avatars.githubusercontent.com/u/10052470?s=460&u=7f2304cb929d1de703856717af86324c66728f3a&v=4'
+authorTwitter: alexmarlette
+tags: [announcement]
+---
+
+## Hello React Native Community,
+
+In May 2020 Facebook was the first company to take the [GAAD pledge](https://diamond.la/GAADPledge/), by doing so they committed to making accessibility a core part of the React Native open source project. Since May, Facebook has spent that time thoughtfully reviewing and documenting accessibility gaps within React Native. So far the gap analysis has surfaced 90 issues, all of which have been translated to Github issues on [React Native's Github](https://github.com/facebook/react-native/projects/15).
+
+Overall, we found that React Native APIs provide strong support for accessibility. However, we also found many core components do not yet fully utilize platform accessibility APIs and support is missing for some platform specific features.
+
+With knowledge of where the gaps exist, work can begin on closing them, the enthusiasm and diversity of contributors have always played a critical role in the development of React Native and whether a seasoned or new contributor, we're asking you to join in making React Native more accessible.
+
+To recognize contributors for their effort, when an accessibility issue is closed and attached to a pull request, contributors will get a shout out on Twitter from our community manager. Contributors whose pull requests are accepted into the codebase will be highlighted in our monthly issues update on Medium.
+
+Please join us in making React Native more accessible for everyone.
+
+### How you can help:
+
+- New contributors should read the [contribution guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md) and browse the list of 46 [good first issues](https://github.com/facebook/react-native/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22+label%3AAccessibility) in the React Native Github
+
+- Seasoned contributors should visit [the project page for Improved React Native Accessibility](https://github.com/facebook/react-native/projects/15) to see the Github issues that need their knowledge of React Native
+
+- Technical writers interested in updating React Native's documentation to reflect the accessibility gaps being closed should visit the [React Native Docs](https://github.com/facebook/react-native-website#-overview)
+
+- Share this initiative with anyone who may be able to help!
+
+[Twitter](https://twitter.com/alexmarlette) | [Facebook](https://www.facebook.com/React-Native-Open-Source-Accessibility-Community-Manager-102732258549941)
```

Comment:
Your Twitter is already linked in the post header so I think there is no need to duplicate it on the bottom, the post is not that long. 🙂  

Also I'm not sure about the purpose of the FB page link, maybe you can put it inside of the message section or "how to help" bullets with some description?

Reply:
I appreciate the feedback, I do want to leave in the direct link to Twitter but putting those links into the "how to help" section is a great addition. The Facebook link is mainly to meet people wherever they spend time, there are some RN communities on FB so we have that page to make it easy for people who don't spend time on twitter to stay updated.